src/inspector/io_agent.cc

```diff
@@ -0,0 +1,50 @@
+#include "io_agent.h"
+#include "crdtp/dispatch.h"
+
+namespace node {
+namespace inspector {
+namespace protocol {
```

Comment:
Nit... since we're finally up on c++20, we can start using the more condensed `namespace node::inspector::protocol {` syntax where appropriate.